### PR TITLE
Basic setup for the repository.

### PR DESCRIPTION
**Pull request by zpChris**
Construct basic repository, with index.html, script.js, style.css, servlets, JUnit testing, and the pom.xml and appengine-web.xml files.

### CODE REVIEW

**Comment by zpChris** [comment]
Sorry about the three commits, I accidentally committed the initial changes on the master branch, so I had to revert that.

**Comment by zpChris** [comment]
Initially, I accidentally copied the original `target` folder (which, if I understand correctly, is the folder that **builds** the site). I refreshed this folder by deleting it, then re-building / running the site on the local port. Those old files from my portfolio should be deleted now.

**Comment by zpChris** [comment]
I requested approval from both Eshika and Adam for first CL (rather than just one person) so we can better understand the process.

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        <version>2.2.0</version>
+        <configuration>
+          <!-- TODO: set project ID. -->
+          <deploy.projectId>offroad-pod-step-2020</deploy.projectId>
+          <deploy.version>1</deploy.version>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+</project>
```
Comment (adamfweidman) [inline pom.xml:59]: nit: add a line

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+  line-height: 1.1;
+}
```
Comment (adamfweidman) [inline src/main/webapp/style.css:6]: nit: add a line

File: src/test/java/com/google/sps/AuthServletTest.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+@RunWith(JUnit4.class)
+public final class AuthServletTest {
+
+  @Test
+  public void testSetup() {
+    Assert.assertTrue(true);
+  }
+
+}
```
Comment (adamfweidman) [inline src/test/java/com/google/sps/AuthServletTest.java:30]: nit: add a line

File: pom.xml (hunk 1 above)
Comment (adamfweidman) [inline pom.xml:59]: nit: add a line

File: src/main/webapp/style.css (hunk 2 above)
Comment (adamfweidman) [inline src/main/webapp/style.css:6]: nit: add a line

File: src/test/java/com/google/sps/AuthServletTest.java (hunk 3 above)
Comment (adamfweidman) [inline src/test/java/com/google/sps/AuthServletTest.java:30]: nit: add a line

File: target/portfolio-1/style.css [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+  line-height: 1.1;
+}
```
Comment (adamfweidman) [inline target/portfolio-1/style.css:6]: nit: add a line

File: target/surefire-reports/TEST-com.google.sps.AuthServletTest.xml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    <property name="java.vendor" value="Oracle Corporation"/>
+    <property name="maven.home" value="/opt/maven"/>
+    <property name="file.separator" value="/"/>
+    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
+    <property name="sun.cpu.endian" value="little"/>
+    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
+    <property name="sun.cpu.isalist" value=""/>
+  </properties>
+  <testcase classname="com.google.sps.AuthServletTest" name="testSetup" time="0.004"/>
+</testsuite>
```
Comment (adamfweidman) [inline target/surefire-reports/TEST-com.google.sps.AuthServletTest.xml:65]: nit: add a line

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  <build>
+    <plugins>
+      <!-- Provides `mvn package appengine:run` for local testing
+           and `mvn package appengine:deploy` for deploying. -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>2.2.0</version>
+        <configuration>
+          <!-- TODO: set project ID. -->
```
Comment (adamfweidman) [inline pom.xml:52]: nit: take out the "TODO"

**Comment by zpChris** [comment]
Sorry for the excessive commits -- since I added the `.gitignore` *after* adding the `target` folder, I had to delete the `target` folder and re-add it in order for the `.gitignore` file to have the desired effect. Now, you should only see the relevant files!

File: pom.xml [hunk 7]
```diff
@@ -0,0 +1,58 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.sps</groupId>
+  <artifactId>portfolio</artifactId>
```
Comment (tristan-jh) [inline pom.xml:7]: I'm not sure if this matters but maybe capstone instead of portfolio?

Comment (zpChris) [reply pom.xml]: Done!

**Pull request merged**